db/migrate/20220130154129_rspo_foreign_key.rb

```diff
@@ -0,0 +1,6 @@
+class RspoForeignKey < ActiveRecord::Migration[7.0]
+  def change
+    add_index :institution_types, [:rspo_institution_type_id], :unique => true
```

Comment:
As long as your branch was not deployed yet, you can just rollback last migration locally and edit it